### PR TITLE
[SD-3353] - Ingest routing schedule is not updated

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

File: server/apps/rules/routing_rules.py [hunk 1]
```diff
@@ -261,6 +261,8 @@ def _adjust_for_empty_schedules(self, routing_scheme):
             schedule = rule.get('schedule')
             if schedule and (set(schedule.keys()) == {'time_zone'}):
                 rule['schedule'] = None
+            elif 'time_zone' not in schedule.keys():
```
Comment (sivakuna-aap) [inline server/apps/rules/routing_rules.py]: This will fail if `schedule` is `None`

Comment (akintolga) [inline server/apps/rules/routing_rules.py]: Schedule can be none, then this rule won't be active as `_get_scheduled_routing_rules` will not return that rule. In that case time_zone is meaningless.

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

**Pull request merged**